googleMapsApiModule.py

```diff
@@ -2,6 +2,7 @@
 import webbrowser
 import config
 import mysql.connector
+from config import key
```

Comment:
This looks redundant. :disappointed_relieved: Since we already use `import config`, we don't need this here. And if use this line to import the necessary variable, it should be used as `key` and not `config.key`

Reply:
@chttrjeankr is correct. Please correct this.